### ISSUE TITLE
Mislabeled active WAN (Multi-WAN failover)

### ISSUE THREAD

**Issue by bmatthewshea** [opnsense/core#6943]
**Describe the bug**

The bug appears to only be COSMETIC:
I setup a multi WAN and group for failover. I expect the active WAN to show as active. It does not.

*The failover function works fine for me other than the mislabeling.*

**To Reproduce**

1. Create a 2 WAN group
2. Set your primary to _Tier 1_. Set failover to _Tier 2_.
3. Verify GW utilized by checking public IP from a network host that's using the WAN group as it's gateway.

**Expected behavior**

Show gateway currently being used as '(active)'.

**Screenshots**

Gateways (WAN_DHCP is the one 'active' not WAN_DHCP2):
![](https://i.imgur.com/cbDcTFB.png)

WAN group:
![](https://i.imgur.com/z9AkDj6.png)

**Environment**

Type 	opnsense
Version 	23.7.6
Architecture 	amd64

**Comment by fichtner** [comment]
@bmatthewshea while it's true that the GUI shows the desired active one it's also true that this is for default gateway switching specifically which you may not have enabled? This or the reload does not go through as desired.


Cheers,
Franco

**Comment by bmatthewshea** [comment]
Thanks for reply.

It does *not* show the 'desired' active one. That's the problem. It is true it shows active on one of them but that is *not* the "active" one (meaning the gateway being used currently).

I am not sure what setting you mean by "default gateway switching" or what it has to do with showing what gateway is "active"? Nor why the gateway would be labeled in a misleading way?

To me "active" means it's active. Right? Or, am I completing misunderstanding it's meaning?

Everything was reloaded/rebooted etc. Same result.

**Comment by fichtner** [comment]
It's showing the system default route that goes through that specific gateway and not the result of the failover group. You can check from System: Routes: Status. Both gateways probably have the same priority... so you can prioritize the tier 1 gateway in your case in the gateway settings under System: Gateways: Single by lowering the value. The default is 255 so setting the tier 1 to 254 should be enough.

But keep in mind this is only cosmetic for routed traffic as it still uses the failover group as per firewall rules (and the firewall decides about that, not the routing table). Only your local firewall traffic will be affected slightly by the change of a default route going out the other tier then.


Cheers,
Franco

**Comment by AdSchellevis** [comment]
https://docs.opnsense.org/manual/gateways.html#default-gateways

**Comment by bmatthewshea** [comment]
Thanks @fichtner & @AdSchellevis - yeah I worked out which gateway to favor by using weight before I wrote OP. Was strictly trying to figure out why active displays on a given GW. 

And thought I had read that doc page. I -do- see the explainer for GW switching and 'active' now, though.

I think I somewhat understand now. I still think 'active' is completely misleading when using failover/redundancy. Maybe another word?
I also think (and apparently [google](https://www.google.com/search?q=what+is+an+upstream+gateway+opnsense+pfsense) does, too) the 'mark as upstream gateway' needs some additional deep explaining in docs. 

Closing.
